src/resources/views/crud/form_content.blade.php

```diff
@@ -22,6 +22,17 @@
 
     <!-- CRUD FORM CONTENT - crud_fields_styles stack -->
     @stack('crud_fields_styles')
+
+    {{-- Temporary fix on 4.1 --}}
```

Comment:
I think this is what should be removed in: #3823 and #3839 is what should be added now in 4.2 since people would need to republish assets and would get the fix there.